ui/site/src/lpv.ts

```diff
@@ -4,16 +4,42 @@ import { Opts } from 'lichess-pgn-viewer/interfaces';
 import { text as xhrText } from 'common/xhr';
 
 export function autostart() {
+  const lpvs = new Map<Element, Element>();
+
   $('.lpv--autostart').each(function (this: HTMLElement) {
-    loadCssPath('lpv').then(() => {
-      Lpv(this, {
-        pgn: this.dataset['pgn']!.replace(/<br>/g, '\n'),
-        orientation: this.dataset['orientation'] as Color | undefined,
-        lichess: location.origin,
-        initialPly: this.dataset['ply'] as number | 'last',
-      });
+    lpvs.set(this.parentElement!, this);
+    Lpv(this, {
+      pgn: this.dataset['pgn']!.replace(/<br>/g, '\n'),
+      orientation: this.dataset['orientation'] as Color | undefined,
+      lichess: location.origin,
+      initialPly: this.dataset['ply'] as number | 'last',
     });
   });
+  if (lpvs.size == 0) return;
+
+  function bindKeyGoto(key: string, goto: string): void {
+    window.Mousetrap.bind(key, () => keyboardTarget()?.dispatchEvent(new Event(goto)));
+  }
+
+  function keyboardTarget(): Element | undefined {
```

Comment:
I don't understand this, how is the target selected?

Reply:
If there are one or more lpvs in the viewport, an event is dispatched to whichever one has the maximum vertical extent onscreen.